source/eitri/ed_handler.py

```diff
@@ -204,23 +199,17 @@ def generate_nav(
     :param docker_cities: the Docker for the "cities" binary
     :param output_file: the name of the output file (usually with extension ".nav.lz4")
     :param ed_component_path: the path of the folder containing all the binaries for data ("*2ed" and "ed2nav")
+    :param cities_file: the path to the file loaded by "cities" (usually a *.osm.pbf)
     :param cities_exec_path: the path of the folder containing the "cities" binary
-    :param import_cities: the path to the directory containing the data for the "ed2nav" binary
     """
     cities_db_params = docker_cities.get_db_params()
     update_db(cities_db_params, ALEMBIC_PATH_CITIES)
 
     ed_db_params = docker_ed.get_db_params()  # type: docker_wrapper.DbParams
     update_db(ed_db_params, ALEMBIC_PATH_ED)
 
-    if import_cities:
-        if not os.path.exists(import_cities):
-            raise Exception('Error: impossible to find {}, exiting'.format(import_cities))
-
-        load_cities(import_cities, cities_db_params, cities_exec_path)
-
-    if not os.path.exists(data_dir):
```

Comment:
This check is already done here : https://github.com/CanalTP/navitia/pull/2852/files/5a86c50102bcdd7e16e42e697f49d72f4d078bbe?file-filters%5B%5D=.py&hide-deleted-files=true#diff-7b35b6684c81d9c6225c1fda49e7d26bR132

It will be display at the beginning not during the execution. So at this point, we have to assume the value is always good.

Reply:
dead link :(

Reply:
Oups, new link : https://github.com/CanalTP/navitia/pull/2852/files#diff-7b35b6684c81d9c6225c1fda49e7d26bR128